src/main/flight/navigation_rewrite_multicopter.c

```diff
@@ -481,13 +481,22 @@ bool isMulticopterLandingDetected(uint32_t * landingTimer, bool * hasHadSomeVelo
     // check if we are moving horizontally
     bool horizontalMovement = sqrtf(sq(posControl.actualState.vel.V.X) + sq(posControl.actualState.vel.V.Y)) > 100.0f;
 
-    // Throttle should be low enough
+    // We have likely landed if throttle is 40 units below average descend throttle
     // We use rcCommandAdjustedThrottle to keep track of NAV corrected throttle (isLandingDetected is executed
     // from processRx() and rcCommand at that moment holds rc input, not adjusted values from NAV core)
-    bool minimalThrust = rcCommandAdjustedThrottle < posControl.navConfig->mc_min_fly_throttle;
-    
+    *landingThrSamples += 1;
+    *landingThrSum += rcCommandAdjustedThrottle;
+    bool minimalThrust= rcCommandAdjustedThrottle < *landingThrSum / *landingThrSamples - 40;
```

Comment:
Help wanted!
If the landingThr-variables are unsigned integers, i get this warning:
```
./src/main/flight/navigation_rewrite_multicopter.c:489:52: 
warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     bool minimalThrust= rcCommandAdjustedThrottle < *landingThrSum / *landingThrSamples - 40;
                                                    ^
```
I could solve that by using this code:
`bool minimalThrust= rcCommandAdjustedThrottle < (int16_t)(*landingThrSum / *landingThrSamples - 40);`
or just use signed intergers for landingThr-variables. @digitalentity how would you like it?

Also, a int16_t/uint16_t is only good for 10/20 minutes of sample counting, should i upgrade to 32 bit integer?